src/app/pages/ledger/connect-ledger/connect-ledger.layout.tsx

```diff
@@ -0,0 +1,50 @@
+import { Box, Stack } from '@stacks/ui';
+
+import { Divider } from '@app/components/divider';
+import { BaseDrawer } from '@app/components/drawer';
+import { Link } from '@app/components/link';
+import { PrimaryButton } from '@app/components/primary-button';
+import { Caption } from '@app/components/typography';
+import ConnectLedger from '@assets/images/onboarding/ledger/connect-ledger.png';
+// import { WarningLabel } from '@app/components/warning-label';
+
+import { LookingForLedger } from './components/looking-for-ledger';
+import { ConnectLedgerTitle } from '../components/connect-ledger-title';
+
+interface ConnectLedgerLayoutProps {
+  isLookingForLedger: boolean;
+  onCancelConnectLedger(): void;
+  onConnectLedger(): void;
+}
+export function ConnectLedgerLayout(props: ConnectLedgerLayoutProps) {
+  const { isLookingForLedger, onCancelConnectLedger, onConnectLedger } = props;
+
+  return (
+    <BaseDrawer title={<Box />} isShowing onClose={onCancelConnectLedger}>
+      <Stack alignItems="center" pb="loose" px="loose" spacing="loose" textAlign="center">
+        <Stack spacing="extra-loose" width="331px">
+          {/* TODO: Implement warning with actual ledger integration */}
+          {/* <WarningLabel>Before proceeding, close Ledger Live if it’s open</WarningLabel> */}
+          <Box mt="tight">
+            <img src={ConnectLedger} />
+          </Box>
+        </Stack>
+        <ConnectLedgerTitle />
+        {isLookingForLedger ? (
+          <LookingForLedger />
+        ) : (
+          <>
+            <PrimaryButton height="40px" onClick={onConnectLedger}>
+              Connect
+            </PrimaryButton>
+            <Divider />
+            <Box width="250px">
```

Comment:
why does this have a fixed width? I am sure I am missing something since if I remove it nothing changes.

Reply:
Yeah, I think this is my mistake, thx for catching. I will remove it.